<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>dialog_confirmar</class>
    <widget class="QDialog" name="dialog_confirmar">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>160</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>400</width>
                <height>160</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>400</width>
                <height>160</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Mensaje</string>
        </property>
        <widget class="QLabel" name="label_confirmar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>30</y>
                    <width>401</width>
                    <height>40</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>13</pointsize>
                </font>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>80</x>
                    <y>110</y>
                    <width>262</width>
                    <height>27</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                    <number>60</number>
                </property>
                <item>
                    <widget class="QPushButton" name="boton_cancelar">
                        <property name="minimumSize">
                            <size>
                                <width>100</width>
                                <height>20</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>45</width>
                                <height>45</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Cancelar</string>
                        </property>
                        <property name="autoDefault">
                            <bool>false</bool>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="boton_aceptar">
                        <property name="minimumSize">
                            <size>
                                <width>100</width>
                                <height>20</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>45</width>
                                <height>45</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Aceptar</string>
                        </property>
                        <property name="autoDefault">
                            <bool>false</bool>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources>
        <include location="recursos/imagenes.qrc"/>
    </resources>
    <connections/>
</ui>
